src/win32/kerberos_sspi.cc

```diff
@@ -452,11 +452,11 @@ static sspi_result* sspi_error_result(DWORD errCode, const SEC_CHAR* msg) {
 
     sspi_result* result = (sspi_result*)malloc(sizeof(sspi_result));
     result->code = AUTH_GSS_ERROR;
-    result->message = (char*)malloc(sizeof(char) * 1024 + 2);
```

Comment:
`sizeof(char)` is always 1 byte right? I thought some windows systems have it as 2. There's docs [here](https://docs.microsoft.com/en-us/cpp/cpp/fundamental-types-cpp?view=msvc-160#character-types) that reference char being upgraded to an `int` so maybe this buffer was 4x the size it should've been?

Reply:
> `sizeof(char)` is always 1 byte right? I

Yes, `sizeof(char)` is always 1, by definition :+1:

> I thought some windows systems have it as 2.

There is a type `TCHAR` on Windows that translates to `wchar_t` (2 bytes) or `char` (1 byte) depending on how the application is being compiled, but that doesn’t affect `char` – maybe that’s what you were thinking of?

> There's docs here that reference char being upgraded to an int so maybe this buffer was 4x the size it should've been?

That’s about type promotion, which doesn’t affect `sizeof()`.